### DISCUSSION TITLE
Github Account compromised and then suspended

### DISCUSSION THREAD

**Discussion by THEPROPHECY55** [community/community#136703] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

On August 5th, my account with the username "PROPHECY55" was compromised, which led to it being flagged and suspended by GitHub. Since then, I’ve been unable to regain access to my account. I reached out to GitHub Support, but I haven’t received any response, and it’s been almost a month now. Unfortunately, I’m also unable to submit multiple inquiries due to the one-ticket limit. My GitHub account is crucial for my university studies, as I rely on it for educational purposes and have important content stored there. I'm unsure of what to do next, as waiting for a reply hasn’t worked. What can I do they seem to ignore me completely?

**Comment by ferid333** [comment]
I'm sorry to hear about your situation. Here are a few steps you can take:

1. **Follow Up on Your Support Ticket:** Try replying to your existing support ticket to bump it up in the queue. Sometimes adding more details or stressing the urgency can help.

2. **Contact GitHub on Social Media:** Reach out to GitHub’s support team via their official Twitter account or other social media platforms. Public mentions can sometimes get quicker responses.

3. **Ask Your University for Help:** Since your GitHub account is crucial for your studies, consider asking your university’s IT department or faculty to contact GitHub on your behalf. Sometimes, institutions can expedite support requests.

4. **Seek Community Help:** Post about your issue on relevant forums or GitHub’s community pages. There might be others who have faced similar issues and can offer advice.

5. **Backup Your Content:** Once you regain access, make sure to back up your important content elsewhere to avoid this kind of issue in the future.

Hopefully, one of these steps will help you get your account back sooner.

**Comment by ebndev** [comment]
Hi @THEPROPHECY55,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
